### DISCUSSION TITLE
Dolphinscheduler无法创建带有Kerberos认证的Hive数据源

### DISCUSSION THREAD

**Discussion by xuhuihihi** [apache/dolphinscheduler#15613] [Q&A]
安装的Dolphinscheduler版本: 3.2.1

我需要添加带有Kerberos认证的Hive数据源，但在页面上添加Hive数据源时发现没法选择kerberos认证，principal和keytab等信息没法填写，该如何解决？

Hive数据源页面如下图：
![161708583938_ pic](https://github.com/apache/dolphinscheduler/assets/3378186/0dab6ca4-b499-4138-b3b6-f465bd1b59ec)

此外我尝试了在页面上的"用户名"处填写principal，以及在"jdbc连接参数"处填写了，如下图
![image](https://github.com/apache/dolphinscheduler/assets/3378186/53d500c5-f712-41d8-86ef-b9869561ff8c)
查看api-server日志有如下报错：
......
Caused by: org.apache.thrift.transport.TTransportException: Peer indicated failure: Unsupported mechanism type PLAIN
......
org.apache.dolphinscheduler.api.controller.DataSourceController#connectDataSource[2 args] Meet a ServiceException: 测试数据源连接失败
......

**Comment by fightBoxing** [comment]
你要先在common.prperties中开启kerberos的认证，页面才会有kerebros的相关信息

**Comment by 329009099** [comment]
想知道搞定了没有啊?
